MapboxSceneKit/TerrainNode.swift

```diff
@@ -79,6 +79,22 @@ open class TerrainNode: SCNNode {
         self.init(southWestCorner: CLLocation(latitude: minLat, longitude: minLon),
                   northEastCorner: CLLocation(latitude: maxLat, longitude: maxLon))
     }
+    
+    @objc public convenience init(lat: CLLocationDegrees, lon: CLLocationDegrees, distanceInKm: Double ) {
```

Comment:
Would it be better to maybe have the center coordinate as a `CLLocation` rather than having it as latitude and longitude?